src/pages/About.mdx

```diff
@@ -73,10 +65,11 @@ Many respiratory viruses such as influenza, common cold viruses (including other
 ## Transmission reduction
 
 The tool allows one to explore temporal variation in the reduction of transmission by infection control measures.
-This is implemented as a curve through time that can be dragged by the mouse to modify the assumed transmission. The curve is read out and used to change the transmission relative to the base line parameters for $R_0$ and seasonality.
+This is implemented as a curve through time that can be dragged by the mouse to modify the assumed transmission. The curve is read out and used to change the transmission relative to the base line parameters for $R_0$ and seasonality. 
+
```

Comment:
Ohhh, this is outdated. 
"Control measures can be specified as time intervals with start and end date during which transmission is reduced by a certain amount."

Reply:
Is a better alternative to remove line 76 (68 in the edit) entirely, and replace it with "Control measures can be specified as time intervals with start and end date during which transmission is reduced by a certain amount."?

Reply:
yes! that is what I meant.